src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  padding: 0;
```

Comment:
don't see any sense to add 'padding: 0' for the html, as html tag doesn't have any paddings